src/modules/UI/scenes/WalletList/style.js

```diff
@@ -224,13 +224,13 @@ export const styles = {
     height: 22,
     width: 22,
     marginRight: 5,
-    resizeMode: Image.resizeMode.contain,
+    resizeMode: 'contain',
```

Comment:
why did this change from an imported string to a hard coded string?

Reply:
New RN removed the constant. Have to use the hard coded string now. Not my choice.